docs/ops/generation/Eye_9.md

```diff
@@ -0,0 +1,170 @@
+## Eye <a name="Eye"></a> {#openvino_docs_ops_generation_Eye_9}
+
+**Versioned name**: *Eye-9*
+
+**Category**: *Generation*
+
+**Short description**: *Eye* operation generates shift matrix or a batch of matrices.
```

Comment:
Shift matrix is an common mathematical term. If it may come to misunderstanding, we can add explanation in brackets.
http://www.ee.imperial.ac.uk/hp/staff/dmb/matrix/special.html#Shift_Matrix